src/sentry/integrations/slack/actions/notification.py

```diff
@@ -109,7 +109,9 @@ def send_notification(event: GroupEvent, futures: Sequence[RuleFuture]) -> None:
 
             client = SlackClient(integration_id=integration.id)
             try:
-                client.post("/chat.postMessage", data=payload, timeout=5)
+                client.post(
```

Comment:
should we add this to metric alerts flow as well?

Reply:
does the metric alert message use blocks?

Reply:
i actually don't think it uses block kit, looking at the class

Reply:
i'm okay isolating to just issue alerts for now